react/tic-tac-toe/src/redux/game/actions.js

```diff
@@ -1,13 +1,16 @@
 import matchesService from '~services/MatchesService';
 
+import localStorageService from '~services/LocalStorageService';
+
 import { SubmissionError } from 'redux-form';
 
 import ERRORS from '~constants/errors';
 
 export const actions = {
   MATCHES: '@@GAME/MATCHES',
   MATCHES_SUCCESS: '@@GAME/MATCHES_SUCCESS',
-  MATCHES_FAILURE: '@@GAME/MATCHES_FAILURE'
+  MATCHES_FAILURE: '@@GAME/MATCHES_FAILURE',
+  GAME_SETTIGNS: '@@GAME/GAME_SETTIGNS'
```

Comment:
Try to make actions as descriptive and simple as possible. What about?
```suggestion
  SET_SETTINGS: '@@GAME/SET_SETTINGS'
```